### PR TITLE
#4417 Shabondi image ought to show the ohara version by entering command "-v"

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4417)
I have test this PR on my local :)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: bin/start-service.sh [hunk 1]
```diff
@@ -44,6 +44,7 @@ else
   if [ "$service" == "configurator" ]; then
     CLASS="oharastream.ohara.configurator.Configurator"
   elif [ "$service" == "shabondi" ]; then
+    # TODO: this main class does not exist (see https://github.com/oharastream/ohara/issues/4422)
     CLASS="oharastream.ohara.shabondi.Boot"
```
Comment (vitojeng) [inline bin/start-service.sh]: 目前朝向曝露 classname 出來, 因此未來會採用直接傳 class name 進來的方式

Comment (chia7712) [reply bin/start-service.sh]: @vitojeng Is there already a ticket? Also, feel free to take over #4422

Comment (vitojeng) [reply bin/start-service.sh]: @chia7712 Already take over, thanks for the reminder :)

Comment (vitojeng) [reply bin/start-service.sh]: > Is there already a ticket?

@chia7712 This task include in my current job, so I not create a issue only for this.

**Pull request merged**